tests/test_ndr.py

```diff
@@ -124,18 +124,12 @@ def test_crit_len_0(self):
         # use predefined directory so test can clean up files during teardown
         args = NDRTests.generate_base_args(self.workspace_dir)
         new_table_path = os.path.join(self.workspace_dir, 'table_c_len_0.csv')
-        with open(new_table_path, 'w') as target_file:

```

Comment:
Rewrote as a pandas operation to more reliably update a specific column, by name, instead of trying to update by index be mutating each line.